src/table/interfaces.tsx

```diff
@@ -321,25 +318,20 @@ export namespace TableProps {
     validation?: (item: T, value: Optional<V>) => Optional<string>;
   }
 
-  export type ColumnEditConfig<ItemType, ValueType> =
-    | {
-        editConfig?: undefined;
-        cell: CellRenderer<ItemType>;
-      }
-    | {
-        editConfig: EditConfig<ItemType, ValueType>;
-        cell: EditableCellRenderer<ItemType, ValueType>;
-      };
-
-  export type ColumnDefinition<ItemType, ValueType = any> = {
+  export interface ColumnEditConfig<ItemType> {
```

Comment:
Any reason to keep it as a separate interface and not merge into `ColumnDefinition`?

Reply:
No reason, other than the extracted type. But if we remove that we can combine this with ColumnDefinition